sample_factory/cfg/cfg.py

```diff
@@ -544,6 +544,15 @@ def add_default_env_args(p: ArgumentParser):
     p.add_argument(
         "--pixel_format", default="CHW", type=str, help="PyTorch expects CHW by default, Ray & TensorFlow expect HWC"
     )
+    p.add_argument(
+        "--multiply_frameskip", default=True, type=str2bool, help="Whether to multiply frameskip for training steps"
```

Comment:
Let's rename this to "summaries_use_frameskip" and put it into "# reporting summaries" section in cfg.py

Help string should be: "Whether to multiply training steps by frameskip when recording summaries, FPS, etc. When this flag is set to True, x-axis for all summaries corresponds to the total number of simulated steps, i.e. with frameskip=4 the x-axis value of 4 million will correspond to 1 million frames observed by the policy."

Reply:
Ok, I'll do that